README.md

```diff
@@ -1,234 +1,248 @@
+  - [👥 Authors ](#-authors-)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [❓ FAQ (OPTIONAL) ](#-faq-optional-)
+  - [📝 License ](#-license-)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Blog App <a name="about-project"></a>
+
+**Blog in Rails App** empowers the end-users to start a new blog-post, comment and like existing content.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+> So you want to lear more about the tech we have used for this project?
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li><a href="https://rubyonrails.org/">Ruby on Rails</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **Make use for migrations features from Active Record**
+- **Include Tables for User, Comment, Post, and Like**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+Note: We are assuming you have pre-installed ruby, WSL, ubuntu and rails!
+
+### Prerequisites
+
+In order to run this project you need:
+
+first at all, install rails:
+
+```sh
+ gem install rails
+```
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```sh
+  cd my-folder
+git@github.com:cancelei/blog-in-rails.git
+```
+
+### Install
+
+Install this project with:
+
+```sh
+  cd blog-app
+  bundle install
+```
+
+Setting up the Data Base:
+
+```sh
+  rails db:create
+  rails db:migrate
+  rails db:seed
+```
+
+### Usage
+
+To run the project, execute the following command:
+
+```sh
+  rails server
+```
+
+### Run tests
+
+<!-- Test are under construction, but we expect to have them completed soon! -->
+
+Before running tests, ensure you've set up the test database:
+
+```sh
+bin/rails db:migrate RAILS_ENV=test
+```
+
+**Note:** Given that the aim of this project is to test the application's UI, to see if it works without side effects. We have therefore had to modify the contents of certain files, which could have the effect of falsifying certain `Unit` tests. 
+We therefore recommend that, rather than running all the tests, you only test the `features` folder and run 
+
+```sh
+bundle exec rspec ./spec/features
+
+```
+
+### Deployment
+
+The deployment will be available soon !
+
+<!--
+Example:
+You can deploy this project using:
+
+```sh
+
+```
+ -->
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Glauber Bannwart**
+
+- GitHub: [@cancelei](https://github.com/cancelei)
+- LinkedIn: [Glauber Bannwart](https://www.linkedin.com/in/gbannwart/)
+

```

Comment:
- [ ] As per this [requirement](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/intergration_spec_and_n_1_project.md#:~:text=on%20each%20project.-,You%20should%20ask%20for%20a%20review%20and%20submit%20this%20activity%20individually.,-Remind%20me%20about), This project milestone suppose to be peer-programming and that is what you all did, but also every one of the team members supposes to submit their own project with own GitHub repo individually and all team members suppose to participate on each others project.

- [ ] [Important notice] I didn't invalidate this PR and continue with my review because you are the owner of the repo. The other submissions by your team members are supposed to be invalidated, not approved as the previous code reviewers did above.